### PR TITLE
People management analytics

### PR DESCRIPTION
**Pull request by oguzkocer**
Relevant #3901. This PR adds the analytics events for people management. It follows the [iOS counterpart.](https://github.com/wordpress-mobile/WordPress-iOS/pull/5430) Thanks @jleandroperez!

P.S: I feel the `AnalyticsTracker` enum is a bit disordered. It's hard to decide where to put the event name and decide on the correct naming. It'd be great if we can maybe order it alphabetically and set some rules. Just my 2 cents.

/cc @astralbodies & @nbradbury 

Needs review: @daniloercoli

### CODE REVIEW

**Comment by daniloercoli** [comment]
> P.S: I feel the AnalyticsTracker enum is a bit disordered. It's hard to decide where to put the event name and decide on the correct naming.

We can reorder the list if that helps. I think we've grouped items by functionalities.

Also, it' s not important the name you give to the enum item, the important part is the name given to the event, that should follow nosara rules.

File: WordPress/src/main/java/org/wordpress/android/ui/ActivityLauncher.java [hunk 1]
```diff
@@ -120,7 +120,7 @@ public static void viewCurrentBlogThemes(Context context) {
     public static void viewCurrentBlogPeople(Context context) {
         Intent intent = new Intent(context, PeopleManagementActivity.class);
         slideInFromRight(context, intent);
-        // TODO: add -> AnalyticsTracker.track(AnalyticsTracker.Stat.OPENED_PEOPLE_MANAGEMENT);
+        AnalyticsUtils.trackWithCurrentBlogDetails(AnalyticsTracker.Stat.OPENED_PEOPLE);
```
Comment (daniloercoli) [inline WordPress/src/main/java/org/wordpress/android/ui/ActivityLauncher.java]: Can we use the old name for this enum `OPENED_PEOPLE_MANAGEMENT`. It feels more clear to me what's used for.

Comment (oguzkocer) [inline WordPress/src/main/java/org/wordpress/android/ui/ActivityLauncher.java]: Hah, that was just a reminder comment I've added for myself, but sure we can 👍

**Comment by daniloercoli** [comment]
:shipit:

**Pull request merged**